README.md

```diff
@@ -209,6 +209,7 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager         |
 |------------------------|----------------------|----------------------|
+| v0.38.0                | v1.20 to v1.22       | v1.4.0 to v1.6.1     |
```

Comment:
Given that we are not actively testing with 1.4.0, should we just specify a concrete version of cert-manager from now and on?